Makefile

```diff
@@ -80,7 +79,7 @@ pkg/minikube/cluster/assets.go: out/localkube $(GOPATH)/bin/go-bindata deploy/is
 
 $(GOPATH)/bin/go-bindata:
 	$(MKGOPATH)
-	go get github.com/jteeuwen/go-bindata/...
+	GOBIN=$(GOPATH)/bin go get github.com/jteeuwen/go-bindata/...
```

Comment:
Instead of moving gobin for this one execution.. why don't you just change line 78 to point to $(GOBIN) if set? It seems like putting the binary in a different directory than expected by the user might cause the user other problems.